### DISCUSSION TITLE
3/21(화) 옵셔널

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#22] [General]
오늘은 옵셔널에 대한 개념을 공부하면 될 것 같습니다. 옵셔널은 진짜 진짜 중요하고 tableView를 사용하던 collectionView를 사용하던 등등 여러가지에 많이 사용도 되고 응용도되기 때문에 잘 알고 가야합니다. 😍

**Comment by Eunho0922** [comment]
넵
